src/prefect/orion/models/workers.py

```diff
@@ -53,7 +53,7 @@ async def create_work_pool(
         session=session,
         work_pool_id=pool.id,
         work_pool_queue=schemas.actions.WorkPoolQueueCreate(
-            name="Default Queue", description="The worker pool's default queue."
+            name="default-queue", description="The worker pool's default queue."
```

Comment:
Do we want to name the default queue just `default`? I think @znicholasbrown brought up that including queue in the name is redundant if we already know it's a queue in our sync on Friday.

Reply:
ah great call - yea i'll update!